transports/webrtc/src/udp_mux.rs

```diff
@@ -0,0 +1,415 @@
+// MIT License
+//
+// Copyright (c) 2021 WebRTC.rs
```

Comment:
Can you signal from where this code was copied?

Reply:
https://github.com/webrtc-rs/ice/tree/main/src/udp_mux. I will clarify.

Reply:
Do we need to copy it? Can we not just depend on in instead?

Reply:
> Do we need to copy it? Can we not just depend on in instead?

It may be possible to make `udp_mux/socket_addr_ext.rs` and `udp_mux/udp_mux_conn.rs` public in the original repo, in which case we'll be able to reuse them. `udp_mux.rs` is a modified version, which has to stay.

Reply:
> `udp_mux.rs` is a modified version, which has to stay.

Okay! Can we document that at the top, also perhaps what exactly is different and why we can't use the original code?